docs/sidebars.yaml

```diff
@@ -1,13 +1,15 @@
 redoc:
-  - group: Quickstart
+  - label: Quickstart
+  - page: redoc/quickstart.md
+  - group: Deployment
     expanded: false
-    page: redoc/quickstart/intro.md
+    page: redoc/deployment/intro.md
```

Comment:
issue: These URLs have already been indexed by search engines.  We would need to set up redirects if we want to change the URLs. Use the [redirectFrom in the front matter](https://redoc.ly/docs/developer-portal/guides/redirects/).

Reply:
ok, I have added the redirects